source/appModules/searchui.py

```diff
@@ -27,7 +36,7 @@ def chooseNVDAObjectOverlayClasses(self,obj,clsList):
 			except ValueError:
 				pass
 		elif isinstance(obj, UIA):
-			if obj.UIAElement.cachedAutomationID == "SearchTextBox":
+			if obj.UIAAutomationId == "SearchTextBox":

```

Comment:
Perhaps this is only ever true in post 18363 builds?

Reply:
Can you verify whether this change is applicable to earlier builds? It may need the same windows version check.